webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html

```diff
@@ -343,35 +343,31 @@ <h2>{{ report.product }} {{ report.version }} Crash Report [@ {{ report.signatur
                             {# Personal user info #}
 
                             {% if request.user.has_perm('crashstats.view_pii') or your_crash %}
-                            <tr title="{{ fields_desc['processed_crash.url'] }}">
-                                <th scope="row">URL</th>
-                                <td>
-                                    {% if raw.URL %}
-                                    <a href="{{ raw.URL }}" title="{{ raw.URL }}">{{ raw.URL }}</a> - {{ sensitive_warning(your_crash) }}
-                                    {% endif %}
-                                </td>
-                            </tr>
-                            <tr title="{{ fields_desc['processed_crash.email'] }}">
-                                <th scope="row">Email Address</th>
-                                <td>
-                                    {% if raw.Email %}
-                                    <a href="mailto:{{ raw.Email }}">{{ raw.Email }}</a> - {{ sensitive_warning(your_crash) }}
-                                    {% endif %}
-                                </td>
-                            </tr>
+                                <tr title="{{ fields_desc['processed_crash.url'] }}">
+                                    <th scope="row">URL</th>
+                                    <td>
+                                        {% if raw.URL %}
+                                        <a href="{{ raw.URL }}" title="{{ raw.URL }}">{{ raw.URL }}</a> - {{ sensitive_warning(your_crash) }}
+                                        {% endif %}
+                                    </td>
+                                </tr>
+                                <tr title="{{ fields_desc['processed_crash.email'] }}">
+                                    <th scope="row">Email Address</th>
+                                    <td>
+                                        {% if raw.Email %}
+                                        <a href="mailto:{{ raw.Email }}">{{ raw.Email }}</a> - {{ sensitive_warning(your_crash) }}
+                                        {% endif %}
+                                    </td>
+                                </tr>
+                                <tr title="{{ fields_desc['processed_crash.user_comments'] }}">
+                                    <th scope="row">User Comments</th>
+                                    <td>
+                                        {% if report.user_comments %}
+                                        {{ report.user_comments | linebreaks }} - {{ sensitive_warning(your_crash) }}
+                                        {% endif %}
+                                    </td>
+                                </tr>
```

Comment:
Previously, two of these fields were under the "must have pii" check. Now all three are. I also fixed the indentation because it was hard to read.